R/as_mapper.R

```diff
@@ -109,10 +109,11 @@ find_extract_default <- function(.null, .default) {
 
 plucker <- function(i, default) {
   x <- NULL # supress global variables check NOTE
+  i <- as.list(i)
 
   new_function(
```

Comment:
hmm... Why are we using `new_function()` here?

Reply:
I presume it's so we get readable source code?

Reply:
Ah yes probably.